.github/workflows/cpp-post-commit.yaml

```diff
@@ -41,7 +41,7 @@ jobs:
         run: tar -xvf ttm_${{ matrix.runner-info.arch }}.tar
       - uses: ./.github/actions/install-python-deps
       - name: ${{ matrix.test-group.name }} tests
-        timeout-minutes: 15
+        timeout-minutes: 30
```

Comment:
@tt-rkim we won't merge this to main, this is just temp to run some gtests using FD out of eth cores on CI